src/styles/cards.scss

```diff
@@ -0,0 +1,22 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+  width: 1024px;
```

Comment:
The fixed width of 1024px for `.cards` might not be responsive on smaller screens. Consider using a percentage-based width or media queries to ensure the layout adapts to different screen sizes.